SmartDeviceLink/SDLManager.h

```diff
@@ -38,7 +38,7 @@ typedef void (^SDLManagerReadyBlock)(BOOL success, NSError *_Nullable error);
 /**
  *  The current HMI level of the running app.
  */
-@property (copy, nonatomic, readonly) SDLHMILevel hmiLevel;
+@property (copy, nonatomic, readonly, nullable) SDLHMILevel hmiLevel;
```

Comment:
I am getting this warning in the *.m* file: `Nullability specifier 'nonnull' conflicts with existing specifier '_Nullable'`.

It should be `- (nullable SDLHMILevel)hmiLevel {...}`

Reply:
Fixed